src/gulpfile.babel.js

```diff
@@ -100,6 +100,7 @@ const reload = done => {
 
 /**
  * Task: `styles`.
+ * @param {function} done callback to indicate the task is done.
```

Comment:
The `@param` comments should be at the end of the function comment with one extra space above them.